src/components/Footer.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { FilterType } from '../enums/enums';
+
+interface FooterProps {
+  incompleteCount: number;
+  filter: FilterType;
+  onFilterChange: (filter: FilterType) => void;
+  todosCompleted: boolean;
+  onClearCompleted: () => void;
+}
+
+export const Footer: React.FC<FooterProps> = ({
+  incompleteCount,
+  filter,
+  onFilterChange,
+  todosCompleted,
+  onClearCompleted,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {incompleteCount} items left
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={`filter__link ${filter === 'all' ? 'selected' : ''}`}
+          data-cy="FilterLinkAll"
+          onClick={() => onFilterChange('all')}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={`filter__link ${filter === 'active' ? 'selected' : ''}`}
+          data-cy="FilterLinkActive"
+          onClick={() => onFilterChange('active')}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={`filter__link ${filter === 'completed' ? 'selected' : ''}`}
+          data-cy="FilterLinkCompleted"
+          onClick={() => onFilterChange('completed')}
+        >
+          Completed
+        </a>
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method